src/ext/respToRedisLoader.c

```diff
@@ -123,7 +128,10 @@ static int redisLoaderWritev(void *context, struct iovec *iov, int count, int st
                                     "Failed to write socket. Exceeded EAGAIN retry limit");
                     return 1;
                 }
-                usleep(1000 * retries); /* Backoff and Retries  */
+
+                /* sleep 1msec */
+                struct timespec req = {0, 1000 * 1000}, rem;
+                nanosleep(&req, &rem);
```

Comment:
sleeps are always evil!!!
why do we have this sleep?
for a plain EAGAIN, we don't need to sleep.
for EWOULDBLOCK maybe we don't want to stay in the loop at all?

Reply:
Ok I will remove the sleep.

EAGAIN & EWOULDBLOCK are actually the same on modern linux machines:
https://man7.org/linux/man-pages/man3/errno.3.html

Reply:
ohh, i was confusing EAGAIN with EINTER.
then i think we want to loop here only on EINTER, and we want to exit if we get EGAIN or EWOULDBLOCK.